src/pages/workspace/accounting/qbo/export/QuickbooksCompanyCardExpenseAccountPage.tsx

```diff
@@ -0,0 +1,96 @@
+            <FeatureEnabledAccessOrNotFoundWrapper
+                policyID={policyID}
+                featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+            >
+                <ScreenWrapper
+                    includeSafeAreaPaddingBottom={false}
+                    testID={QuickbooksCompanyCardExpenseAccountPage.displayName}
+                >
+                    <HeaderWithBackButton title={translate('workspace.qbo.exportCompany')} />
+                    <ScrollView contentContainerStyle={styles.pb2}>
+                        <Text style={[styles.ph5, styles.pb5]}>{translate('workspace.qbo.exportCompanyCardsDescription')}</Text>
+                        <OfflineWithFeedback pendingAction={pendingFields?.exportCompanyCard}>
+                            <MenuItemWithTopDescription
+                                title={exportCompanyCard ? translate(`workspace.qbo.${exportCompanyCard}`) : undefined}
+                                description={translate('workspace.qbo.exportCompany')}
+                                error={errorFields?.exportCompanyCard ? translate('common.genericErrorMessage') : undefined}
+                                onPress={() => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_COMPANY_CARD_EXPENSE_SELECT.getRoute(policyID))}
+                                brickRoadIndicator={errorFields?.exportCompanyCard ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
+                                shouldShowRightIcon
+                            />
+                        </OfflineWithFeedback>
+                        {!!exportCompanyCard && (
+                            <Text style={[styles.ph5, styles.mutedNormalTextLabel, styles.pt1, styles.pb2]}>{translate(`workspace.qbo.${exportCompanyCard}Description`)}</Text>
+                        )}
+                        {isVendorSelected && (
+                            <>
+                                <OfflineWithFeedback pendingAction={pendingFields?.exportAccountPayable}>
+                                    <MenuItemWithTopDescription
+                                        title={exportAccountPayable}
+                                        description={translate('workspace.qbo.accountsPayable')}
+                                        error={errorFields?.exportAccountPayable ? translate('common.genericErrorMessage') : undefined}
+                                        onPress={() => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_COMPANY_CARD_EXPENSE_ACCOUNT_PAYABLE_SELECT.getRoute(policyID))}
+                                        brickRoadIndicator={errorFields?.exportAccountPayable ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
+                                        shouldShowRightIcon
+                                    />
+                                </OfflineWithFeedback>
+                                <ToggleSettingOptionRow
```

Comment:
The addition of this toggle turned out to have a small UI difference from what design was expecting, leading the following issue:
- https://github.com/Expensify/App/issues/43730

basically missing `shouldPlaceSubtitleBelowSwitch` (true) which was rendering the toggle in the center of both `title` and `subtitle` container, instead of rendering the `subtitle` underneath the [title - toggle] container.

Reply:
thank you for fixing it!